config.yaml

```diff
@@ -48,14 +48,14 @@ options:
       type: boolean
       default: False
       description: >
-        This option enables auto update for plugins. The update will check for 
-        timestamp modifications on the .hpi files in the configured plugins-site
-        and will download plugins if newer files are available.
+        This option enables auto update for plugins. The routine will check for
+        new versions available based on update center's info, will update them
```

Comment:
Is it worth capitalising "Update Center's" here, or explaining a bit more about what "update center" is?

Reply:
I've capitalized the word and a better description of it can be found in the update-center config's description.

Reply:
f0ce544